_posts/2016-06-11-oord16.md

```diff
@@ -21,8 +21,8 @@ page: 1747-1756
 order: 1747
 cycles: false
 author:
-- given: Aaron Van
-  family: Oord
+- given: Aäron
```

Comment:
Thanks Kento!

The right format should be to introduce as a prefix. Can you update so that it is in the form: 

prefix: van den
family: Oord

Reply:
Thank you for your suggestion! I've fixed it.